lnrpc/walletrpc/psbt.go

```diff
@@ -57,7 +57,7 @@ func lockInputs(w lnwallet.WalletController,
 		}
 
 		// Get the details about this outpoint.
-		utxo, err := w.FetchInputInfo(&lock.Outpoint)
+		utxo, err := w.FetchOutpointInfo(&lock.Outpoint)
```

Comment:
Would be nice to re-arrange the commits to have this one before the actual changes in behavior to the locking.

Reply:
So basically the last 4 commits should be the firs ones IMO.

Reply:
Will do once the `btcwallet` PR is in

Reply:
Dependent PR now merged.